packages/playground/src/weblets/tf_openwebui.vue

```diff
@@ -0,0 +1,278 @@
+        </input-tooltip>
+
+        <TfSelectionDetails
+          :filters="{
+            ipv4,
+            ipv6,
+            hasGPU,
+            certified,
+            dedicated,
+            rentedBy,
+            cpu: solution?.cpu,
+            ssdDisks: disks.map(disk => disk.size),
+            solutionDisk: solution?.disk,
+            memory: solution?.memory,
+            rootFilesystemSize,
+            planetary,
+            mycelium,
+            wireguard,
+          }"
+          require-domain
+          v-model="selectionDetails"
+        />
+
+        
+        <manage-ssh-deployemnt @selected-keys="updateSSHkeyEnv($event)" />
+      </template>
+    </d-tabs>
+
+    <template #footer-actions="{ validateBeforeDeploy }">
+      <v-btn
+        variant="elevated"
+        class="text-primery px-10 py-3 h-auto text-subtitle-1"
+        @click="validateBeforeDeploy(deploy)"
+        text="Deploy"
+      />
+    </template>
+  </weblet-layout>
+</template>
+
+<script lang="ts" setup>
+import { computed, type Ref, ref, watch } from "vue";
+
+import { manual } from "@/utils/manual";
+
+import Networks, { useNetworks } from "../components/networks.vue";
+import { useLayout } from "../components/weblet_layout.vue";
+import { useGrid, useProfileManager } from "../stores";
+import type { solutionFlavor as SolutionFlavor } from "../types";
+import { type Flist, ProjectName } from "../types";
+import { deployVM, type Disk } from "../utils/deploy_vm";
+import { deployGatewayName, getSubdomain, rollbackDeployment } from "../utils/gateway";
+import { normalizeError } from "../utils/helpers";
+import { generateName } from "../utils/strings";
+
+const selectionDetails = ref<SelectionDetails>();
+
+const layout = useLayout();
+const profileManager = useProfileManager();
+const solution = ref() as Ref<SolutionFlavor>;
+const selectedSSHKeys = ref("");
+const name = ref(generateName({ prefix: "oi" }));
+const flist = ref<Flist>({
+  name: "Ubuntu-24.04 Open WebUI Instance",
+  value: "https://hub.grid.tf/tf-official-apps/threefoldtech-ubuntu-24.04_fullvm_oi.flist",
+  entryPoint: "",
+});
+const { ipv4, ipv6, mycelium, planetary, wireguard } = useNetworks();
+const dedicated = ref(false);
+const rentedByMe = ref(false);
+const rentedBy = computed(() => (rentedByMe.value ? grid.twinId : undefined));
+const certified = ref(false);
+const disks = ref<Disk[]>([]);
+const hasGPU = ref(false);
+const rootFilesystemSize = computed(() => solution.value?.disk);
+const gridStore = useGrid();
+const grid = gridStore.client as GridClient;
+
+watch(
+  [dedicated, rentedByMe],
+  ([dedicated, rentedByMe]) => {
+    if (dedicated === false && rentedByMe === false) {
+      hasGPU.value = dedicated;
+    }
+  },
+  { immediate: true },
+);
+
+watch(
+  hasGPU,
+  hasGPU => {
+    if (hasGPU) {
+      dedicated.value = true;
+      rentedByMe.value = true;
+    }
+  },
+  { immediate: true },
+);
+
+function finalize(deployment: any, domain: string) {
```

Comment:
its not using the domain now, please remove the domain here and in the line this function got called

Reply:
@mik-tf small change please make sure to address it before merge